index.js

```diff
@@ -1,59 +1,9 @@
-var async = require('async')
-var dynamoTable = require('./slim')
+const {DynamoTable} = require('./slim');
+const DynamoAdapter = require('./dynamo-adapter');
 
-module.exports = dynamoTable
+// Library is a factory function which also provides the DynamoTable class
+const dynomite = (...args) => new DynamoAdapter(...args);
```

Comment:
dynomite -> dynamite

Reply:
Lol yeah i did name it incorrectly on purpose but i mean, probably I should change it anyway in case